src/utils/bivariate/bivariateLegendUtils.ts

```diff
@@ -89,19 +89,28 @@ export function createBivariateMeta(
   };
 }
 
+// need to ignore first range to avoid getting @ and 0 chars
+const indexTestFunc = <T extends { value: number }>(
```

Comment:
Can we improve naming?

For example
```
const fitsLegendCriteria = <T extends { value: number }>(
  step: T,
  index: number,
  steps: T[],
  targetValue: number
): 
```

Do we need to pass the whole array, or steps count will be enough?